src/language/rule/rule.ts

```diff
@@ -165,6 +165,8 @@ export interface ReplacementJson {
     innerText: string;
 }
 export class Replacement {
+    constructor(readonly start: number, readonly length: number, readonly text: string) {}
```

Comment:
why are these constructors moved? i did not expect them to come first.

Reply:
>tslint.json: use preset "fields-first" for member-ordering rule instead of previous old-style config.
>This seems to match previous config most closely, requiring only 5 changes.

I can recheck if one of the other presets matches more closely, otherwise, this seems the way to go...

Reply:
Yeah this preset is the closest.